### PR TITLE
Osucascades header

### PR DESCRIPTION
**Pull request by emerham**

### CODE REVIEW

**Comment by emerham** [comment]
Adds theme options only available to Admins and DX Administrators to be able to set the Companion Logo

File: logo.svg
Comment (Ruefa) [inline logo.svg:1]: image

**Review by Ruefa — APPROVED** [review]
LGTM github actions already called you out

File: logo.svg
Comment (emerham) [reply logo.svg:1]: yeah the files were just "optimized". Will deal with the failing check on code standards

**Pull request merged**